bt2_values_generator.rb

```diff
@@ -81,19 +89,45 @@ class BTValueCLass::Bool < BTValueCLass::Scalar
     @bt_type_is = 'bt_value_is_bool'
     @bt_return_type = 'bt_bool'
     @bt_default_value = 'BT_FALSE'
+
+    def initialize(name,usr_default_value)
+      bt_type = self.class.instance_variable_get(:@bt_type)
+      if !usr_default_value.nil? and ![true, false].include? usr_default_value
```

Comment:
Please change it, it should be a string and it should be `BT_TRUE` or `BT_FALSE`

Reply:
Just to be sure, does the user will use BT_TRUE or BT_FALSE in params.yaml for bool fields?